std_msgs/Header header 
string[] detected_classes
float32[] scores
sensor_msgs/RegionOfInterest[] boxes
std_msgs/Float32MultiArray features